reddit_liveupdate/activity.py

```diff
@@ -18,51 +19,48 @@
 ACTIVITY_FUZZING_THRESHOLD = 100
 
 
+@baseplate_integration.with_root_span("job.liveupdate_activity")
 def update_activity():
     events = {}
     event_counts = collections.Counter()
-    event_ids = ActiveVisitorsByLiveUpdateEvent._cf.get_range(
-        column_count=1, filter_empty=False)
 
-    for event_id, is_active in event_ids:
-        count = 0
+    for chunk in utils.in_chunks(LiveUpdateEvent._all(), size=100):
+        context_ids = {"LiveUpdateEvent_" + ev._id: ev._id for ev in chunk}
+
+        try:
+            with c.activity_service.retrying() as svc:
+                infos = svc.count_activity_multi(context_ids.keys())
+        except TTransportException:
+            continue
+
+        for context_id, info in infos.iteritems():
+            event_id = context_ids[context_id]
 
-        if is_active:
             try:
-                count = ActiveVisitorsByLiveUpdateEvent.get_count(event_id)
+                LiveUpdateActivityHistoryByEvent.record_activity(
```

Comment:
Do we keep track of this indefinitely?  How often are we adding these?  (Not saying it's bad or should be documented; just for own curiosity).